src/LA/system_solvers_small.cpp

```diff
@@ -353,6 +363,62 @@ backwardSubstitution(U, z, x);
 
 return; };
 
+// -------------------------------------------------------------------------- //
+/*!
+    Solve a linear system using Lapacke_dgesv (partial pivoting
+    and LU factorization). Containers MUST be correctly sized.
+    This routine works on the entire matrix, the number of considered equations
+    is equal to the leading dimension of the matrix, the same is for the r.h.s.
+    Multiple r.h.s. are not allowed. Permutation matrix not provided.
+    This function is intended for solution computation ONLY.
+
+    \param[in] layout matrix layout (possible values are ROW_MAJOR and COL_MAJOR)
+    \param[in,out] A in input the coeffs of square matrix (linear), in output the permuted factors L and U.
```

Comment:
Can we use the same desciption for both the solveLU functions?

Reply:
Almost. See new push, please